src/site/paragraphs/collapsible_panel.drupal.liquid

```diff
@@ -40,15 +41,17 @@
                         aria-controls="{{ item.entityBundle }}-{{ item.entityId }}"
```

Comment:
`aria-controls` would be used on a focusable element with a click handler to communicate that a trigger point (usually a button) is connected to an ID in the DOM

Reply:
unrelated to changes in this PR.